airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/state/StreamManager.kt

```diff
@@ -124,10 +126,11 @@ class DefaultStreamManager(
         return recordCount.get()
     }
 
-    override fun markEndOfStream(): Long {
+    override fun markEndOfStream(complete: Boolean): Long {
         if (markedEndOfStream.getAndSet(true)) {
```

Comment:
(Deleted my previous comment, because I missed that you were still using the `markedEndOfStream` flag here.

I think `complete` is confusing here because we're already using it to mean "processing complete." Maybe call `complete`/`receivedComplete`/`isComplete` something that indicates it's a platform status?

Reply:
Ah fair enough. I'll noodle on the naming a bit.

Reply:
From your suggestions, I think going with `receivedComplete` for now is clear enough, but kind of couples us deeply to the stream status implementation for now. Seems like a reasonable tradeoff.

Reply:
Kotlin was mad at the variable name collision so `receivedStreamCompleteMessage` it is. A little verbose, but no one should get confused!